var/spack/repos/builtin/packages/axel/package.py

```diff
@@ -10,18 +10,15 @@ class Axel(AutotoolsPackage):
     """Axel is a light command line download accelerator for Linux and Unix"""
 
     homepage = "https://github.com/axel-download-accelerator/axel"
-    url      = "https://github.com/axel-download-accelerator/axel/archive/v2.16.1.tar.gz"
+    url      = 'https://github.com/axel-download-accelerator/axel/releases/download/v2.17.10/axel-2.17.10.tar.bz2'
 
-    version('2.16.1', sha256='64529add74df3db828f704b42d4ec3fcdacb8142c84f051f9213637c337e706c')
+    version('2.17.10', sha256='c0d26eba6b94945cd98c5b69ca6df2744639d17bfd49047ef51a8a48f067de10')
+    version('2.16.1', sha256='763066efc61e4f7be2eb59afa049bdbc520837e01c95a78f403e542ad82f2719')
 
-    depends_on('m4', type='build')
     depends_on('autoconf', type='build')
+    depends_on('autoconf-archive', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
     depends_on('pkgconfig', type='build')
-    depends_on('gettext')
+    depends_on('gettext',   type='build', when='@2.17.10:')
```

Comment:
axel@2.16.1 also looks for gettext, why restrict this?

Reply:
Removed, works now! The Issue I had seems to have been fixed by using the release tarball and not running autogen.sh!